File: web_shop/src/main/webapp/js/service/allOrderService.js
// 定义服务层:
app.service("allOrderService",function($http){


    this.fh = function(ids){
        return $http.get("../allorder/fh.do?ids="+ids);
    }
	this.search = function(page,rows,searchEntity){
		return $http.post("../allorder/search.do?pageNum="+page+"&pageSize="+rows,searchEntity);
	}

});

File: pyg_interface/src/main/java/cn/itcast/core/service/AllOrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.entity.BuyerCart;
import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.log.PayLog;
import cn.itcast.core.pojo.order.Order;

import java.util.List;

public interface AllOrderService {

    void fh(Long[] ids);

    PageResult search(Integer pageNum, Integer pageSize, Order order);
}
